src/main/kotlin/dev/kord/x/emoji/DiscordEmoji.kt

```diff
@@ -33,31 +33,40 @@ sealed class DiscordEmoji {
         override val unicode: String
             get() = "$code${tone.unicode}"
 
+        /**
```

Comment:
This could be simplified to 
```kt
        fun isSimilar(other: Any?): Boolean {
            return this === other || (other is Diverse && code == other.code)
        }
```
It'd be easier to keep track of

Reply:
I'll approve however as it's not an urgent matter